### DISCUSSION TITLE
How to correctly get the organic structure information from the Gaussian16 output file for RDKit reading?

### DISCUSSION THREAD

**Discussion by jshan2020-33** [rdkit/rdkit#5306] [Q&A]
How can the organic structure obtained by Gaussian16 optimization be correctly used for RDKit to calculate molecular descriptors?

Try to use GaussView, OpenBabel, Multiwfn to convert the sdf/mol/pbd files, read these files with RDKit, and find that RDKit can only read files generated by OpenBabel. However, using OpenBabel to directly convert the out/log/fch/fchk file into a mol file, there are two organics that cannot be read, and the display type is NoneType after reading. However, the fch/fchk file is first converted to a mol file with GaussView, and then the mol is generated with OpenBabel, but these two compounds can be read. Comparing the mol files generated by the two methods, it is found that the coordinates of xyz are consistent, but the rest of the connection information is different.

Comparison of mol generated in two ways:
![C9D~JYCQ LZR$@AYTAS`$YS](https://user-images.githubusercontent.com/81633044/169272065-f2ab18bb-8014-4327-9f2c-9901ebc333c3.png)

Comparing some organic compounds that can be read by RDKit in the mol files generated by the two methods, the calculated features show that some features are consistent, but there are still some feature values that are quite different.
![image](https://user-images.githubusercontent.com/81633044/169273665-354da840-01a7-4720-be1e-39b83e4b1585.png)

Since the mol files obtained by the two methods are missing for RDKit to generate features, but the feature values generated by the two methods are not completely consistent, it is uncertain whether they can be used in combination. If you can find other better ways, it will be a better choice to use the structure obtained by Gaussian optimization for RDKit to calculate the generated features.

RDKit: 2020.09.1.0
Python: 3.7

**Comment by greglandrum** [comment]
To make sure I understand the question correctly: you have some mol files generated by OpenBabel which the RDKit doesn't like (i.e. it returns None when you try to read them)?
If you share those mol files we might be able to help you understand what is going on here.

**Reply by jshan2020-33** [reply]
Of course, I store the mol files obtained and the original files in both ways in the zip file below. Thank you very much for your attention to this issue.

[RDKit_read_OpenBabel.zip](https://github.com/rdkit/rdkit/files/8787582/RDKit_read_OpenBabel.zip)

RDKit: 2020.09.1
OpenBabel: 3.1.1
GaussView: 6.0.16
Python: 3.7

**Reply by roderick1717** [reply]
May I ask if this problem has been solved? I have also encountered this problem and look forward to your reply.

**Reply by qitianya** [reply]
me too
